code/raspi_sensor_tests/allsensors.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import smbus2
+import bme280
+import serial
+import pynmea2
+from time import time, sleep
+from w1thermsensor import W1ThermSensor
+
+i2c = smbus2.SMBus(1)
+
+ds18b20 = W1ThermSensor()
+BME280_ADDR = 0x76
+BME280_ADDR_IMU = 0x77
+
+gps_port_path = "/dev/serial0"
+gps_serial_port = serial.Serial(gps_port_path, baudrate = 9600, timeout = 0.5)
+
+a = time()
+
+while True:
+	sleep(0.1)
+	if not "-bme280" in sys.argv:
```

Comment:
for now it's good enough, but please get familiar with [argparse](https://docs.python.org/3/howto/argparse.html) :)